sdk/eventhub/event-hubs/review/event-hubs.api.md

```diff
@@ -69,16 +71,19 @@ export class EventDataBatch {
     constructor(context: ConnectionContext, maxSizeInBytes: number, partitionKey?: string);
     readonly batchMessage: Buffer | undefined;
     readonly count: number;
+    // @internal
+    readonly _messageSpanContexts: SpanContext[];
     readonly partitionKey: string | undefined;
     readonly sizeInBytes: number;
-    tryAdd(eventData: EventData): boolean;
+    // Warning: (ae-forgotten-export) The symbol "TryAddOptions" needs to be exported by the entry point index.d.ts
```

Comment:
assuming you noticed this?

Reply:
Yes, I've got an update for this, as well as instrumenting the client APIs :)